src/source/speed.rs

```diff
@@ -1,3 +1,50 @@
+//! Playback Speed control Module.
+//!
+//! The main concept of this module is the [`Speed`] struct, which
+//! encapsulates playback speed controls of the current sink.
+//!
+//! In order to speed up a sink, the speed struct:
+//! - Increases the current sample rate by the given factor
+//! - Updates the total duration function to cover for the new factor by dividing by the factor
+//! - Update the pos function by multiplying the position by the factor
```

Comment:
I think you mean to describe how try_seek works for a sped up source here

Reply:
Done